ext/sst/sst_responder_subcomponent.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 The Regents of the University of California
+// Copyright (c) 2023 The Regents of the University of California
```

Comment:
You've done this all over but you want to do `2021-2023`. What you're saying here is you don't claim copyright for anything prior to 2023.

Fix this in the places you've done this.

Reply:
I'll fix this in the next patch.